posts/2024-04-15-simplifying-nosql-database-integration-with-jakarta-nosql.adoc

```diff
@@ -0,0 +1,418 @@
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Simplifying NoSQL Database Integration with Jakarta NoSQL
+Maximillian Arruda <https://github.com/dearrudam>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+
+// // // // // // // //
+// In the preceding section:
+// Do not insert any blank lines between any of the lines.
+//
+// "open-graph-image" is set to OL logo. Whenever possible update this to a more appropriate/specific image (For example if present a image that is being used in the post). However, it
+// can be left empty which will set it to the default
+//
+// "open-graph-image-alt" is a description of what is in the image (not a caption). When changing "open-graph-image" to
+// a custom picture, you must provide a custom string for "open-graph-image-alt".
+//
+// Replace DESCRIPTION with a short summary (~60 words) of the release (a more succinct version of the first paragraph of the post).
+//
+// If adding image into the post add :
+// -------------------------
+// [.img_border_light]
+// image::img/blog/FILE_NAME[IMAGE CAPTION ,width=70%,align="center"]
+// -------------------------
+// "[.img_border_light]" = This adds a faint grey border around the image to make its edges sharper. Use it around screenshots but not
+// around diagrams. Then double check how it looks.
+// There is also a "[.img_border_dark]" class which tends to work best with screenshots that are taken on dark backgrounds.
+// Change "FILE_NAME" to the name of the image file. Also make sure to put the image into the right folder which is: img/blog
+// change the "IMAGE CAPTION" to a couple words of what the image is
+// // // // // // // //
+
+In the data-driven age of AI and cloud computing, choosing the right data storage solution directly impacts the success of a solution. We currently have two major data storage approaches to rely on: relational and NoSQL databases. Within the Java space, when it comes to working with relational databases, the Java Persistence API (JPA) specification has undoubtedly established itself as a robust and widely adopted API. In contrast, the crowded landscape of NoSQL database options opens a large set of integration options for developers, who instead of relying on standards, often find themselves learning and depending on vendor's proposed solutions.
+
+> "The way you define data storage and code design will either facilitate or hinder the project's maintainability, performance, scalability, and ability to change databases later on."
+
+Fortunately, the Java community adopting NoSQL storage solutions no longer needs to worry about these concerns. The Jakarta NoSQL specification offers a well-defined, standard-based implementation with low learning curve and familiar coding APIs, thanks to its similarities with the widely-adopted JPA specification.
```

Comment:
Maybe consider changing from "implementation with low learning curve" to "implementation with a low learning curve"